src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.task.Task;
+
+/**
+ * An UI component that displays information of a {@code Task}.
+ */
+public class TaskCard extends UiPart<Region> {
+
+    private static final String FXML = "TaskCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">
+     * The issue on TeachingAssistantBuddy level 4</a>
```

Comment:
Is this a potential issue? It does not flow in this wording now but will it fail checks if we change it back to Addressbook?